guides/v2.1/coding-standards/code-standard-php.md

```diff
@@ -27,10 +27,10 @@ This includes references to:
 
 Examples:
 
-{% highlight php startinline=true %}
+```php
```

Comment:
have you verified that this renders properly without the additional inline syntax?

> php?start_inline=1

there are _many_ php code blocks in this PR that could be affected if this is required.

Reply:
I will check them. Seems to work when all spacing/indentation rules are followed.